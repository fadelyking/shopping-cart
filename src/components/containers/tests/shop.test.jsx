import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";
import Shop from "../shop";

const images = [{ 1: 1, 2: 3, 4: 5 }];

describe("Shop Page", () => {
	it("Should render the shop page", (props) => {
		const { container } = render(
			<MemoryRouter>
				<Shop images={images} />
			</MemoryRouter>
		);
		expect(container).toMatchSnapshot();
	});
});
