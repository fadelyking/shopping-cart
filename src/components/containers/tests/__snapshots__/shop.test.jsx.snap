// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Shop Page > Should render the shop page 1`] = `<div />`;
